pkgs/development/python-modules/arrow/default.nix

```diff
@@ -16,7 +16,7 @@ buildPythonPackage rec {
   '';
 
   checkInputs = [ nose chai simplejson ];
-  propagatedBuildInputs = [ dateutil backports_functools_lru_cache ];
+  propagatedBuildInputs = [ dateutil backports_functools_lru_cache_121 ];
```

Comment:
I would rather patch the line `install_requires.append('backports.functools_lru_cache==1.2.1')` from `setup.py` to read `install_requires.append('backports.functools_lru_cache>=1.2.1')`.

Reply:
I've patched the `setup.py` in 340cadd97d9.